lib/ui/natives.dart

```diff
@@ -78,10 +75,16 @@ void _scheduleMicrotask(void Function() callback) native 'ScheduleMicrotask';
 int? _getCallbackHandle(Function closure) native 'GetCallbackHandle';
 Function? _getCallbackFromHandle(int handle) native 'GetCallbackFromHandle';
 
-// Required for gen_snapshot to work correctly.
-int? _isolateId; // ignore: unused_element
```

Comment:
I can find no references to this in the engine or in the dart/sdk. Since it does not have a pragma it is almost certainly getting tree shaken

Reply:
https://github.com/flutter/engine/pull/2762

Reply:
Confirmed this was removed from the Dart SDK in https://dart-review.googlesource.com/c/sdk/+/129742 and is safe to delete